readme_release.md

```diff
@@ -79,5 +82,10 @@ Then the process is less automated (but still, instructions are given):
 	```
 
 # Troubleshooting
-If you run into github's daily limitation, you can easily provide your login/token into the script.
-Search for "rate limit exceeded" in script.
+Note that if the release process fails in the middle, here are a few tips that might help before launching it again:
+
+- check you're on `dev` branch
+- check the branch `release` doesn't exist (might have been created by a previous execution); **or else** remove it
```

Comment:
Are you sure about that one?
To me this is a bug, no?

Reply:
I removed this line ([011c2ac](https://github.com/hove-io/navitia/pull/3768/commits/011c2ace249921a3ef6ef19e74dd544186bbfbfe)). We might come back to it later to identify if it's a bug or not.